src/wallet/wallet_tx_builder.cpp

```diff
@@ -207,9 +432,7 @@ InputSelectionResult WalletTxBuilder::ResolveInputsAndPayments(
 
     // we can only select Orchard addresses if there are sufficient non-Sprout
     // funds to cover the total payments + fee.
```

Comment:
Non-blocking nit: this comment is now out-of-date. We can fix it later.

Reply:
Please add a TODO comment or file an issue, if it isn't fixed now.